sdks/python/scripts/generate_pydoc.sh

```diff
@@ -76,10 +79,15 @@ excluded_patterns=(
     *_pb2.py
     *_test.py
     *_test_common.py
-    # TODO(BEAM-7847): Remove this once doc generation can parse Py3 syntax.
-    *_py3*.py
 )
 
+python_version=`python -V 2>&1`
```

Comment:
This addition could be removed if we assumed this was always running under Python 3.7.

Reply:
Exactly, removed.